### PR TITLE
Sepolia deploy

### PR DESCRIPTION
**Pull request by codenamejason**

### CODE REVIEW

**Comment by ZakGriffith** [comment]
Let's set the defaultNetwork back to localhost.

File: yarn.lock [hunk 1]
```diff
@@ -23237,3 +23257,8 @@ zen-observable@^0.8.0, zen-observable@^0.8.14:
   version "0.8.15"
   resolved "https://registry.yarnpkg.com/zen-observable/-/zen-observable-0.8.15.tgz#96415c512d8e3ffd920afd3889604e30b9eaac15"
   integrity sha512-PQ2PC7R9rslx84ndNBZB/Dkv8V8fZEpk83RLgXtYd0fwUgEjseMn1Dgajh2x6S8QbZAFa9p2qVCEuYZNgve0dQ==
+
+zksync-web3@^0.8.1:
```
Comment (damianmarti) [inline yarn.lock:23261]: @codenamejason why zksync-web3 is here?

Comment (codenamejason) [reply yarn.lock:23261]: I wonder if related to updating this package `@nomiclabs/hardhat-etherscan": "^3.1.5` @damianmarti

Comment (damianmarti) [reply yarn.lock:23261]: > I wonder if related to updating this package `@nomiclabs/hardhat-etherscan": "^3.1.5` @damianmarti

I think that zksync-web3 is not a dependency from @nomiclabs/hardhat-etherscan. Maybe you switch to another branch, run a "yarn install" and the yarn.lock gets updated and then you switch to this new branch with the file modified...

Comment (damianmarti) [reply yarn.lock:23261]: @codenamejason I found where this zksync-web3 stuff come from. It's a dependency from hardhat-deploy. It was updated at https://github.com/scaffold-eth/scaffold-eth/commit/e4afffc7e3d6c2bb575240b4de830703e8925548 by @ZakGriffith but he forgot to commit the updated yarn.lock

File: packages/hardhat/deploy/00_deploy_your_contract.js [hunk 2]
```diff
@@ -64,18 +65,17 @@ module.exports = async ({ getNamedAccounts, deployments, getChainId }) => {
...
-  //       constructorArguments: [],
-  //     });
-  //   }
-  // } catch (error) {
-  //   console.error(error);
-  // }
+  // You can also Verify your contracts with Etherscan here during the deployment process
+  try {
+    if (chainId !== localChainId) {
+      // await run("verify:verify", {
```
Comment (damianmarti) [inline packages/hardhat/deploy/00_deploy_your_contract.js:71]: Why are you commenting the verify?

**Review by damianmarti — APPROVED** [review]

**Pull request merged**